conjure-java/src/main/java/com/palantir/conjure/java/cli/ConjureJavaCli.java

```diff
@@ -223,6 +223,12 @@ public static final class GenerateCommand implements Runnable {
                 description = "Union visitors expose the values of unknowns in addition to their types.")
         private boolean unionsWithUnknownValues;
 
+        @CommandLine.Option(
+                names = "--externalFallbackTypes",
+                defaultValue = "false",
+                description = "Java external type imports are generated using their fallback type.")
+        private boolean externalFallbackTypes;
```

Comment:
This option is kind of like type erasure to the base/fallback type, and is all or nothing right now.

In the future we might want something in-between, for example to erase `com.palantir.CustomThing` to a conjure `string`, while keeping `java.time.ZoneId` as an external import because that class is more widely available.

No action now, but wanted to mention this in case it changes the way the options are arranged now.

Reply:
I did consider something special with java standard-library types, but opted for more of an absolutist view to align with the code we produce for other languages like typescript, golang, and python.

Reply:
That makes sense to me.  If there are special cases for Java->Java, then it makes everything non-Java second class in a way.  We aim to make all supported languages equally supported, so pruning benefits of same-language conjure calls is a way to do that.